apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -42,10 +50,72 @@ const Ratings: React.FC<{ id: number; tutorName: string | null }> = ({
 }) => {
   const { user } = useUser();
   const intl = useFormatMessage();
+  const toast = useToast();
+  const invalidateQuery = useInvalidateQuery();
+  const [deleteDialog, setDeleteDialog] = useState<number | null>(null);
+
+  const [editDialog, setEditDialog] = useState<{
+    id: number;
+    studentName: string;
+    studentId: number;
+    imageUrl: string;
+    feedback: string | null;
+    rating: number;
+  } | null>(null);
+
+  const closeDelete = useCallback(() => {
+    setDeleteDialog(null);
+  }, []);
+
+  const onDeleteError = useCallback(
+    () =>
+      toast.error({
+        title: intl("tutor.rating.delete.error"),
+      }),
+    [intl, toast]
+  );
+
+  const onDeleteSuccess = useCallback(() => {
+    toast.success({
+      title: intl("tutor.rating.delete.success"),
+    });
```

Comment:
No need to show a toast message. Student can see the result of his action already.